RubberduckTests/QuickFixes/RemoveDuplicatedAnnotationQuickFixTests.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System.Linq;
+using System.Threading;
+using NUnit.Framework;
+using Rubberduck.Inspections.Concrete;
+using Rubberduck.Inspections.QuickFixes;
+using RubberduckTests.Mocks;
+
+namespace RubberduckTests.QuickFixes
+{
+    [TestFixture]
+    public class RemoveDuplicatedAnnotationQuickFixTests
+    {
+        [Test]
+        [Category("QuickFixes")]
+        public void RemoveDuplicatedAnnotation_QuickFixWorks_RemoveDuplicate()
+        {
+            const string inputCode = @"
+'@Obsolete
+'@Obsolete
+Public Sub Foo
+End Sub";
+
+            const string expectedCode = @"
+'
```

Comment:
it'd be really cool if the quickfix could remove the leading `'` if all annotations in a line are removed

Reply:
I will try to correct this.